packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -36,6 +38,37 @@ interface Props {
 
 const ProjectForm: FC<Props> = ({ disabled, defaultType }) => (
     <>
+        <Card
+            style={{
+                marginBottom: '20px',
+                display: 'flex',
+                flexDirection: 'row',
+                gap: 20,
+            }}
+            elevation={1}
+        >
+            <div style={{ flex: 1 }}>
+                <div
+                    style={{
+                        marginBottom: 15,
+                    }}
+                >
+                    <H5 style={{ display: 'inline', marginRight: 5 }}>
+                        Base details
```

Comment:
I think `General` or `General settings` is better here

Reply:
Is the name required in onboarding @ZeRego ?

Reply:
Not really, I can set it to be the org name as before.